src/test/java/domain/LottoFactoryTest.java

```diff
@@ -0,0 +1,30 @@
+package domain;
+
+import org.junit.jupiter.api.DisplayName;
+import org.junit.jupiter.api.Test;
+
+import java.util.Arrays;
+import java.util.List;
+import java.util.stream.Collectors;
+
+import static org.assertj.core.api.Assertions.assertThat;
+
+class LottoFactoryTest {
+
+    @DisplayName("로또 생성")
+    @Test
+    void generateLotto() {
+        //given
+        List<LottoNumber> numbers = Arrays.asList(1, 2, 3, 4, 5, 6).stream()
+                .map(LottoNumber::new)
+                .collect(Collectors.toList());
```

Comment:
```suggestion
        List<LottoNumber> numbers = Stream.of(1, 2, 3, 4, 5, 6)
                .map(LottoNumber::new)
                .collect(Collectors.toList());
```
바로 스트림을 사용할 거 라면 stream.of()가 더 나을 거 같아요 :)